src/main/webapp/script.js

```diff
@@ -173,46 +160,147 @@ function getSchoolInfo() {
           (document.getElementById('race-piechart'));
       chart.draw(data, options);
-    }
+}
 
-    function drawGenderChart() {
-      let data = google.visualization.arrayToDataTable([
-        ['Gender', 'Percentage'],
-        ['Men', numMen],
-        ['Women', numWomen],
-      ]);
+/**
+ * Creates Donut Pie Chart displaying gender breakdown.
+ */
+function drawGenderChart(numMen, numWomen) {
+  let data = google.visualization.arrayToDataTable([
+      ['Gender', 'Percentage'],
+      ['Men', numMen],
+      ['Women', numWomen],
+  ]);
 
-      let options = {
-        title: 'Breakdown by Gender',
-        pieHole: 0.4,
-        colors: ['#D4EBD0', '#A4C5C6'],
-      };
+  let options = {
+      title: 'Breakdown by Gender',
+      pieHole: 0.4,
+      colors: ['#D4EBD0', '#A4C5C6'],
+  };
 
-      let chart = new google.visualization.PieChart
-          (document.getElementById('gender-piechart'));
-      chart.draw(data, options);
+  let chart = new google.visualization.PieChart
+      (document.getElementById('gender-piechart'));
+  chart.draw(data, options);
+}
+
+ /**
+ * Finds and returns the main campus from a list of main and satellite campuses
+ * of the same name.
+ * @param {!Array<object>} schools This is the array of school objects returned
+ * from a fetch query.
+ */
+function getMainCampus(schools) {
+  if (schools.length == 1) {
+    return schools[0];
+  } else {
+    main = {}
+    schools.forEach((campus) => {
+        if (campus['school.main_campus'] == 1) {
+          main = campus;
+        }
+    });
+    return main;
     }
-  });
 }
 
-function getLink(schoolName) {
-  return ('https://api.data.gov/ed/collegescorecard/v1/schools.json?' +
-      'api_key=C8Uyh2jQCmfjfKN3qwqwcJOi77c5k3V6zM7cRFgJ&school.name=' + 
-      schoolName + '&fields=id,school.city,school.name,school.state,school' +
-      '.school_url,school.ownership,school.minority_serving.' +
-      'historically_black,latest.admissions.admission_rate.overall,' +
-      'latest.admissions.sat_scores.average.overall,latest.admissions' +
-      '.act_scores.midpoint.cumulative,latest.cost.tuition,latest.cost' +
-      '.avg_net_price,latest.student.size,latest.student.demographics' +
-      '.race_ethnicity.white,latest.student.demographics.race_ethnicity' +
-      '.black,latest.student.demographics.race_ethnicity.hispanic,' +
-      'latest.student.demographics.race_ethnicity.asian,latest' +
-      '.student.demographics.race_ethnicity.aian,latest.student.demographics' +
-      '.race_ethnicity.nhpi,latest.student.demographics.race_ethnicity' +
-      '.two_or_more,latest.student.demographics.race_ethnicity' +
-      '.non_resident_alien,latest.student.demographics.race_ethnicity' +
-      '.unknown,latest.student.demographics.men,latest.student' +
-      '.demographics.women,latest.completion.completion_rate_4yr_100nt,' +
-      'school.main_campus,school.institutional_characteristics.level')
+ /**
```

Comment:
Indent -1. Actually, looks like an issue throughout the file - check all of them.